src/docfx/build/xref/XrefResolver.cs

```diff
@@ -105,6 +105,7 @@ internal class XrefResolver
 
         query = queries.AllKeys.Length == 0 ? "" : "?" + string.Join('&', queries);
         var fileLink = UrlUtility.MergeUrl(xrefSpec.Href, query, fragment);
+        Telemetry.TrackLink("xref");
```

Comment:
This is likely going to have a perf impact similar to markdown element telemetry as we have many many links. We are basically grouping item count of `FileLinkBuilder` so track the aggregated result in `FileLinkBuilder.Build` method.